main.go

```diff
@@ -385,10 +403,22 @@ func schemaFromDump(driver string, ioHelper *ioStreams) (*internal.Conv, error)
 }
 
 func dataFromDump(driver string, config spanner.BatchWriterConfig, ioHelper *ioStreams, client *sp.Client, conv *internal.Conv) (*spanner.BatchWriter, error) {
-	_, err := ioHelper.seekableIn.Seek(0, 0)
-	if err != nil {
-		fmt.Printf("\nCan't seek to start of file (preparation for second pass): %v\n", err)
-		return nil, fmt.Errorf("can't seek to start of file")
+	if !dataOnly {
+		_, err := ioHelper.seekableIn.Seek(0, 0)
+		if err != nil {
+			fmt.Printf("\nCan't seek to start of file (preparation for second pass): %v\n", err)
+			return nil, fmt.Errorf("can't seek to start of file")
+		}
+	} else {
+		// Note: input file is kept seekable to plan for future
```

Comment:
At some point we might need to do a refactor of the way we handle getSeekable to avoid the code duplication here: potentially we could do this right at the top of toSpanner i.e. before we call schemaConv. We could look at the driver and call getSeekable if we have pg_dump or mysqldump.

But let's leave as is for the moment.

Reply:
Yes correct. For now, I will add it as TODO in this block.